__tests__/app.test.js

```diff
@@ -101,4 +101,27 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("GET article comments by id", () => {
+    test("200 + returns array of comments with correct properties and values", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.comments;
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+              })
+            );
```

Comment:
A note about this test - if you were testing an empty array, the test would still pass, which isn't the behavior we want for this article. Because you're testing inside a forEach, if there's nothing to iterate through (empty), then these lines of tests wouldn't have a chance to run.

To avoid this, we need to check the length before the forEach!